telemetry/definitions/commonDefinitions.json

```diff
@@ -982,7 +991,9 @@
                 { "type": "version", "required": false },
                 { "type": "lambdaPackageType", "required": false },
                 { "type": "reason", "required": false },
-                { "type": "eventBridgeSchema", "required": false }
+                { "type": "eventBridgeSchema", "required": false },
+                { "type": "architecture", "required": false },
```

Comment:
Should we just keep `architecture` confined to the VSC Toolkit? It's somewhat counter to the goal of this PR, though it may be less confusing for other Toolkits.

Reply:
Or even just give up on `architecture` entirely as I think it's just specific to VSC. We can fix the data on the backend as-needed.

Reply:
That makes sense. Because it's vscode-only, none of our main dashboards are using this field.